src/Cake.Common/Xml/XmlPokeAliases.cs

```diff
@@ -0,0 +1,177 @@
+﻿using System;
+using System.IO;
+using System.Xml;
+using Cake.Core;
+using Cake.Core.Annotations;
+using Cake.Core.IO;
+
+namespace Cake.Common.Xml
+{
+    /// <summary>
+    /// Contains functionality related to XML XSL transformation.
+    /// </summary>
+    [CakeAliasCategory("XML")]
+    public static class XmlPokeAliases
+    {
+        /// <summary>
+        /// Set the value of, or remove, target nodes.
+        /// </summary>
+        /// <param name="context">The context.</param>
+        /// <param name="filePath">The target file.</param>
+        /// <param name="xpath">The xpath of the nodes to set.</param>
+        /// <param name="value">The value to set too. Leave blank to remove the selected nodes.</param>
+        /// <param name="settings">Additional settings to tweak Xml Poke behavior.</param>
+        [CakeMethodAlias]
```

Comment:
Would be great if we could add an example of how to use the alias with the `example` XML tag.